device/arista/x86_64-arista_720dt_48s/td3x2-a720dt-48s-flex.config.bcm

```diff
@@ -219,30 +219,30 @@ portmap_56.0=64:10
 port_flex_enable.0=1
 port_gmii_mode{25}=1
 port_gmii_mode{41}=1
-port_init_autoneg_1.0=0
-port_init_autoneg_2.0=0
-port_init_autoneg_3.0=0
-port_init_autoneg_4.0=0
-port_init_autoneg_5.0=0
-port_init_autoneg_6.0=0
-port_init_autoneg_7.0=0
-port_init_autoneg_8.0=0
-port_init_autoneg_9.0=0
-port_init_autoneg_10.0=0
-port_init_autoneg_11.0=0
-port_init_autoneg_12.0=0
-port_init_autoneg_13.0=0
-port_init_autoneg_14.0=0
-port_init_autoneg_15.0=0
-port_init_autoneg_16.0=0
-port_init_autoneg_17.0=0
-port_init_autoneg_18.0=0
-port_init_autoneg_19.0=0
-port_init_autoneg_20.0=0
-port_init_autoneg_21.0=0
-port_init_autoneg_22.0=0
-port_init_autoneg_23.0=0
-port_init_autoneg_24.0=0
+port_init_autoneg_1.0=1
```

Comment:
Do we need to set the autoneg to `true` in phy{N}_config.json?

Reply:
From testing the PAI driver it seems that setting the autoneg in the phy*_config.json files is not necessary.